nautobot/ipam/views.py

```diff
@@ -150,7 +148,7 @@ class RouteTargetBulkDeleteView(generic.BulkDeleteView):
 
 
 class RIRListView(generic.ObjectListView):
-    queryset = RIR.objects.annotate(aggregate_count=count_related(Aggregate, "rir"))
+    queryset = RIR.objects.annotate(prefix_count=count_related(Prefix, "rir"))
```

Comment:
Maybe a TODO comment here? Aggregate count is pretty self-explanatory; prefix count is a bit ambiguous as to whether it's only counting top-level prefixes, counting prefixes at any level that have an assigned RIR, or counting all such prefixes and their descendants. We should disambiguate this at some point.

Reply:
I'd like to fix it in this PR. I think the only place it's really exposed to the user is on the RIR serializer:

```json
        {
            "id": "b8c65d37-c37c-4db3-81cd-c00610b6f002",
            "display": "AFRINIC",
            "url": "http://localhost:8080/api/ipam/rirs/b8c65d37-c37c-4db3-81cd-c00610b6f002/",
            "name": "AFRINIC",
            "slug": "afrinic",
            "is_private": false,
            "description": "Performance people west need themselves measure ask. Perhaps fine exactly grow everybody seem second.\nThird perhaps glass article day including. Treatment site recent.",
            "prefix_count": 4123,
            "created": "2023-02-15",
            "last_updated": "2023-02-15T16:53:05.790659Z",
            "notes_url": "http://localhost:8080/api/ipam/rirs/b8c65d37-c37c-4db3-81cd-c00610b6f002/notes/",
            "custom_fields": {}
        },
```

What about `related_prefix_count`?

Reply:
Here at least it is a column in the RIR table in the UI as well. Maybe `assigned_prefix_count`?